reporting/api/src/main/java/org/jboss/windup/reporting/service/ReportService.java

```diff
@@ -18,11 +18,10 @@
 
 /**
  * Convenient search and creation methods for {@link ReportModel}.
- *
  */
 public class ReportService extends GraphService<ReportModel>
 {
-    private static final String REPORTS_DIR = "reports";
+    public static final String REPORTS_DIR = "reports";
```

Comment:
Ok. I can't get the context through @Inject, can I? So I am passing it from the RuleProvider.